app/views/machine.njk

```diff
@@ -102,6 +70,23 @@
 {% endblock %}
 
 {% block js %}
-{#  <script src="/js/vendor/jquery.tablesort.min.js"></script>#}
-{#  <script src="/js/machine.js"></script>#}
-{% endblock %}
+ <script>
+  let backupCalendar = [];
+  {% for week in backupCalendar %}
+    {% for day in week %}
+      cell = {
+        bucket: "{{day.bucket}}",
+        today: "{{day.today}}",
+        backup: "{{day.bucket.backup}}",
+        id: "{{day.id}}",
+        date: "{{day.date}}",
+      }
+      backupCalendar.push(cell)
```

Comment:
You're missing a `;` here, as well as in a few other places:
- Where `cell` is defined
- Where you call the `generateCalendar` function
- In some of the `if` statements in the `generateCalendar` function